src/c++/perf_analyzer/request_rate_worker.cc

```diff
@@ -54,6 +54,18 @@ RequestRateWorker::Infer()
   } while (true);
 }
 
+void
+RequestRateWorker::CreateContexts()
+{
+  // If the model is non-sequence model, use one InferContext to
+  // maintain concurrency for this thread.
```

Comment:
This is why comments can be bad. You can't copy code and the comments and then change just the code. There is no "concurrency" here.

Reply:
removed